tests/v2/e2e/assets/rollout.yaml

```
@@ -13,14 +13,199 @@
+        executions:
+          - name: Insert
+            type: insert
+            mode: unary
+            parallelism: 10
+            num: 60000
+            qps: 3000
+            wait: 2m
+          - mode: unary
+            name: IndexInfo
+            type: index_info
+        name: Insert -> IndexInfo
```

YAMLlint (1.35.1)

[error] duplication of key "name" in mapping

(key-duplicates)